src/transformers/models/siglip/convert_siglip_to_hf.py

```diff
@@ -49,6 +49,8 @@
     "siglip-base-patch16-256-i18n": "/Users/nielsrogge/Documents/SigLIP/webli_i18n_b16_256_66117334.npz",
     # so400m checkpoints
     "siglip-so400m-patch14-384": "/Users/nielsrogge/Documents/SigLIP/webli_en_so400m_384_58765454.npz",
+    "siglip-so400m-patch14-224": "/Users/mervenoyan/Desktop/siglips/webli_en_so400m_224_57633886.npz",
+#    "siglip-so400m-patch16-256": "/Users/mervenoyan/Desktop/siglips/webli_i18n_so400m_16_256_78061115.npz",
```

Comment:
We should have -i18n in the model name, my fault for forgetting this in our colab, updating soon.

Reply:
Updated our colab.

Reply:
To be uncommented?

Reply:
Yep I'm porting it